galsim/roman/roman_psfs.py

```diff
@@ -382,10 +382,11 @@ def _read_aberrations(SCA):
     # use entries 1-22).  The units are waves.
     aberrations = np.zeros((5,23))
     aberrations[:,1:] = dat[:,5:]
-    # Also get the field position.  The file gives it in arcsec with respect to the center, but we
-    # want it in pixels with respect to the corner.
-    x_sca_pos = dat[:,1]/pixel_scale + n_pix/2
-    y_sca_pos = dat[:,2]/pixel_scale + n_pix/2
+    # Also get the field position.  The file gives it in mm with respect to the center, but we
+    # want it in pixels with respect to the corner. The pixel size of the detector is 0.01 mm/pixel
+    pixel_size = 0.01 
```

Comment:
I was wondering if it's worth storing this number in `__init__.py` along with other basic parameters of Roman, so it can be used here and in the test suite without hard coding it as 0.01?  Seems reasonable to keep it with the rest of the basic reference info there.

Reply:
Yeah. I have stored the value as `pixel_scale_mm` in the roman `__init__.py`